src/page/SequencePage.test.js

```diff
@@ -14,6 +14,18 @@ test('checks, if input for timecode is there', () => {
   expect(placeholder).toBeVisible()
 })
 
+test('checks, if "Tor" Tag is there', () => {
```

Comment:
You could try using "describe() and it()" to cluster your tests and make it more readable, if you like.